addons/website_event/views/event_templates.xml

```diff
@@ -412,19 +412,21 @@
 <!-- Event - Description -->
 <template id="event_description_full" name="Event Description">
     <t t-call="website_event.event_details">
-        <section class="bg-200">
-            <div class="container">
-                <div class="row no-gutters mt-n5 pb-5">
+        <section class="bg-200 mt-n5">
+            <div class="container rounded overflow-hidden">
+                <div class="row no-gutters mt-n4">
                     <!-- Description -->
-                    <div id="o_wevent_event_main_col" class="col-lg-8 bg-white rounded-left p-5 shadow-sm">
+                    <div id="o_wevent_event_main_col" class="col-lg-8 bg-white p-5 shadow-sm">
                         <span t-field="event.description" itemprop="description"/>
                     </div>
-                    <div class="col-lg-4 bg-light rounded-right shadow-sm d-print-none">
+                    <div class="col-lg-4 bg-light shadow-sm d-print-none">
                         <!-- Date & Time -->
                         <div class="o_wevent_sidebar_block">
                             <h6 class="o_wevent_sidebar_title">Date &amp; Time</h6>
-                            <span t-field="event.with_context(tz=event.date_tz).date_begin" t-options="{'date_only': 'true', 'format': 'EEEE'}"/>
-                            <h4 class="my-1" t-field="event.with_context(tz=event.date_tz).date_begin" t-options="{'date_only': 'true', 'format': 'long'}" itemprop="startDate" t-att-datetime="event.date_begin"/>
+                            <div class="d-flex align-items-center">
```

Comment:
`align-items-center` doesn't seem to change anything and is not on event end_date.